contributors/guide/community-expectations.md

```diff
@@ -45,6 +45,8 @@ respectful Kubernetes community.
 
 ## Expectations of reviewers: Review latency
 
+As Github has introduced the status in the user account, now the bot uses this status for auto-assigning reviewers, for example if a reviewer has a status set to "Busy" so the bot will not assign them new PRs for review .
```

Comment:
To stay consistent with how the next paragraph is worded, how about rephrasing this as what reviewers are expected to do instead? :)

So adding something like `Reviewers are expected to set their [user status](https://help.github.com/en/articles/personalizing-your-profile#setting-a-status) to "busy" if they are unavailable to review for sometime. If they are unavailable for a longer period of time, they are expected to remove themselves from the OWNERS file and potentially nominate someone else.` after the next paragraph would be good IMO.